ultraplot/tests/test_geographic.py

```diff
@@ -758,3 +760,68 @@ def test_inset_axes_geographic():
         latlim=(20, 30),
     )
     return fig
+
+
+def test_tick_toggler():
+    fig, ax = uplt.subplots(proj="cyl")
+    for pos in "left right top bottom".split():
+        if pos in "left right".split():
+            ax.format(latlabels=pos)
+        else:
+            ax.format(lonlabels=pos)
+        ax.set_title(f"Toggle {pos} labels")
+        # Check if the labels are on
+        # For cartopy backend labelleft can contain
+        # False or x or y
+        label = f"label{pos}"
+        assert ax[0]._is_ticklabel_on(label) != False
+        ax[0]._toggle_gridliner_labels(**{label: False})
+        assert ax[0]._is_ticklabel_on(label) != True
+    uplt.close(fig)
+
+
+@pytest.mark.mpl_image_compare
+def test_sharing_cartopy_with_colorbar():
+
+    def are_labels_on(ax, which=["top", "bottom", "right", "left"]) -> tuple[bool]:
```

Comment:
```suggestion
    def are_labels_on(ax, which=("top", "bottom", "right", "left")) -> tuple[bool]:
```

Using mutable objects as function defaults is a big anti-pattern in python and will causes unexpected side effects.